### ISSUE TITLE
issue undefined emoji

### ISSUE THREAD

**Issue by eMerzh** [iansinnott/zazu-emoji#3]
hi, 
i've update zazu to the latest version (not sure it's related...)
and i noticed that some emoji are "undefined" like the worker .
(App Environment: production
App Version: 0.5.1
Electron Version: 1.3.14
Node Version: 6.5.0
Chrome Version: 52.0.2743.82)
 
<img width="602" alt="capture d ecran 2017-03-26 a 20 48 51" src="https://cloud.githubusercontent.com/assets/177003/24334158/b51e739e-1265-11e7-8fef-bbab72432481.png">

**Comment by iansinnott** [comment]
Hm, I wasn't able to reproduce this. I upgraded to the latest zazu (0.5.1). Maybe if you give me more information about your system I could help. Try viewing the "About zazu" page from the menu bar dropdown (if on a Mac). 

<img width="179" alt="screen shot 2017-03-26 at 12 31 20 pm" src="https://cloud.githubusercontent.com/assets/3154865/24334513/4fd1d07a-1220-11e7-8b3a-81a4fe73e5b6.png">
<img width="601" alt="screen shot 2017-03-26 at 12 30 40 pm" src="https://cloud.githubusercontent.com/assets/3154865/24334514/4fd1e696-1220-11e7-85c2-04afade350dc.png">

**NOTE:** The broken images on some of the search results are actually expected. When I built this plugin zazu only supported images for the icon previews, which meant that I had to find an image of every emoji. I wasn't able to find them all, so some of the emoji are without image unfortunately.
